jenkins/Jenkinsfile-blossom.premerge

```diff
@@ -390,7 +390,7 @@ pipeline {
                     }
                 } // end of DB runtime 7.3
 
-                stage('DB runtime 8.2') {
+                stage('DB runtime 9.1') {
```

Comment:
please also update stage expression,

db_build && major_ver >= 21 && minor_ver >= 6
==>
db_build && major_ver >= 21 && **minor_ver >= 12**

as we added spark312db shim later for 21.12 or later

Reply:
fixed